lib/relationships.js

```diff
@@ -49,32 +60,97 @@ export function parseEnv(input) {
   }, new Map());
 }
 
+/**
+ * @param {string} repoName
+ * @returns {string}
+ */
 export function getParentName(repoName) {
   return get(REPO_RELATIONSHIP_MAP.get(repoName), 'parent');
 }
 
+/**
+ * @param {string} repoName
+ * @returns {ChildType[]}
+ */
 export function getChildren(repoName) {
   return get(REPO_RELATIONSHIP_MAP.get(repoName), 'children') || [];
 }
 
+/**
+ * @param {string} repoName
+ * @returns {string[]}
+ */
 export function getChildrenNames(repoName) {
   return getChildren(repoName).map(child => child.name);
 }
 
+/**
+ * @param {string} parentRepoName
```

Comment:
Why are we using jsdoc instead of flow here?

Reply:
It started off as just being quicker than adding flow (since vscode supports out of the box) but kinda expanded to the whole repo. I can switch to flow

Reply:
Feel free to do that latter - doesn't need to be included in this PR